snippets/xslt-snippets.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
XSLT completion support based on XSD schema for XSL is implemented and provides those kind of completion items, what is the benefit of those snippet?

Reply:
To reduce keystrokes required to create elements and attributes with prefilled attributes.

Reply:
@EmilySeville7cfg Perhaps you're not ware that there's an [XSLT/XPath language extension](https://marketplace.visualstudio.com/items?itemName=deltaxml.xslt-xpath) for VS Code? 

This extension has context-aware auto-completion snippets for all XSLT 3.0 (and the proposed XSLT 4.0 instructions) - the snippets include attributes and child-elements where required. As well as XML validation, XPath expressions, variables, functions etc. are also validated.